c/src/ml-api-inference-internal.c

```diff
@@ -297,6 +297,44 @@ __ml_validate_model_file (const char *const *model,
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Internal helper function to read the string from specific ini file.
```

Comment:
Please mention that the returned value should be freed by the caller in @return .